src/libPMacc/include/particles/tasks/TaskSendParticlesExchange.hpp

```diff
@@ -113,6 +113,15 @@ namespace PMacc
         virtual ~TaskSendParticlesExchange()
         {
             notify(this->myId, RECVFINISHED, NULL);
+            if(retryCounter != 0)
+            {
+                std::cerr << "Send buffer for particle species " <<
+                    ParBase::FrameType::getName() <<
+                    " is to small (max particles per send operation:" << maxSize <<
+                    ", direction:" << exchange <<
+                    ", retried:" << retryCounter <<
```

Comment:
I'd also shorten the message as much as possible while you are on it. So repeated output doesn't produce that much file size (Requested that due to 13GB text file sizes...)
E.g.: "Sendbuffer for species X to small (Max. particles: Y, Dir: Z, Retries: Q)"